src/box/memtx_tx.c

```diff
@@ -38,6 +38,11 @@
 #include "schema_def.h"
 #include "small/mempool.h"
 
+enum
+{
+        MEMTX_TX_DONTSPLIT_PART_COUNT = UINT32_MAX
```

Comment:
Comment explaining what this thing is - is a must.

Reply:
Fixed.